requirements.txt

```diff
@@ -12,6 +12,4 @@ pywin32 ; platform_system == 'Windows'
 certifi ; platform_system == 'Windows'
 
 # VSCode robotframework-lsp requires robotframework directly in venv
-robotframework==6.0.2
-# robotframework >= 6.1 is only supported with robotframwork-lsp >= 1.11.0,
-# but https://github.com/robocorp/robotframework-lsp/issues/947
+robotframework==7.3.2
```

Comment:
FTR, I don't like that this is in `requirements.txt`.  It ends up on dist.plone.org/release, so lots of projects that directly use the `requirements.txt` will install `robotframework` needlessly.

I would keep it here in 6.1, but would be in favour of removing it in coredev 6.2.

Mind you, I don't do much with robot tests, and I don't think I have the VS code extension for it installed.